### PR TITLE
Fix lods of effects

### PR DESCRIPTION
**Pull request by MadMaxFAF**
This branch is for bugfixes, general improvements, and new features. If your change is designed to alter balance, please make 
sure your changes are rebased onto [development/balance] (https://github.com/FAForever/fa/tree/development/balance), and target that branch with your PR.

### CODE REVIEW

File: lua/aeonweapons.lua [hunk 1]
```diff
@@ -330,6 +330,34 @@ AANDepthChargeBombWeapon = Class(DefaultProjectileWeapon) {
     end,
 }
 
+AANDepthChargeBombWeapon02 = Class(DefaultProjectileWeapon) {
```
Comment (Garanas) [inline lua/aeonweapons.lua:333]: Where did this come from?

**Comment by Garanas** [comment]
I'll more properly review it later this week.

File: lua/aeonweapons.lua (hunk 1 above)
Comment (MadMaxFAF) [reply lua/aeonweapons.lua:333]: that effect was used on a destroyer and t1 arty as well as strat bombers so i split it to have different lods

Comment (Garanas) [reply lua/aeonweapons.lua:333]: Good thinking!

Comment (MadMaxFAF) [reply lua/aeonweapons.lua:333]: Antiairmuzzle02 i added an 01 for the lower lods and had to split the aeon projectile as it was being used by both

File: effects/emitters/cybran_lra_vent_effect_01_emit.bp [hunk 2]
```diff
@@ -1,3 +1,4 @@
+used by urb2302
```
Comment (Garanas) [inline effects/emitters/cybran_lra_vent_effect_01_emit.bp:1]: A comment is missing here, but I'll catch that when I review it tonight

File: effects/emitters/antimatter_hit_01_emit.bp [hunk 3]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = true,
-	LODCutoff = -1.00,
+	LODCutoff = 300,
```
Comment (Garanas) [inline effects/emitters/antimatter_hit_01_emit.bp:14]: These feel a bit low, given that it is used by the t3 uef artillery - what if we move it up to 500?

File: effects/emitters/bomb_hit_fire_01_emit.bp [hunk 4]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 100.00,
```
Comment (Garanas) [inline effects/emitters/bomb_hit_fire_01_emit.bp:14]: This should default to 300

File: effects/emitters/cannon_artillery_flash_01_emit.bp [hunk 5]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 100.00,
```
Comment (Garanas) [inline effects/emitters/cannon_artillery_flash_01_emit.bp:14]: I think this one can be set to 200

File: effects/emitters/cannon_artillery_muzzle_flash_01_emit.bp [hunk 6]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = true,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 100.00,
```
Comment (Garanas) [inline effects/emitters/cannon_artillery_muzzle_flash_01_emit.bp:14]: I think this one can be set to 200

File: effects/emitters/cybran_lra_ground_effect_01_emit.bp [hunk 7]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = true,
-	LODCutoff = -1.00,
+	LODCutoff = 150.00,
```
Comment (Garanas) [inline effects/emitters/cybran_lra_ground_effect_01_emit.bp:14]: ground effect (the shockwave) can be a bit higher, say 250

File: effects/emitters/default_muzzle_flash_02_emit.bp [hunk 8]
```diff
@@ -1,3 +1,24 @@
+-- AANChronoTorpedoWeapon
```
Comment (Garanas) [inline effects/emitters/default_muzzle_flash_02_emit.bp:1]: So many!!

File: effects/emitters/cybran_tactical_missile_launch_01_emit.bp [hunk 9]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 100.00,
```
Comment (Garanas) [inline effects/emitters/cybran_tactical_missile_launch_01_emit.bp:14]: Given how unusual this happens, setting it a bit higher to say 160 is fine.

File: effects/emitters/destruction_explosion_concussion_ring_03_emit.bp [hunk 10]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = true,
-	LODCutoff = -1.00,
+	LODCutoff = -1.00, -- experimental death weapon always visable
```
Comment (Garanas) [inline effects/emitters/destruction_explosion_concussion_ring_03_emit.bp:14]: All good.

File: effects/emitters/destruction_explosion_fire_shadow_01_emit.bp [hunk 11]
```diff
@@ -1,3 +1,4 @@
+-- used by FireShadowMed01 FireShadowLrg01
```
Comment (Garanas) [inline effects/emitters/destruction_explosion_fire_shadow_01_emit.bp:1]: image

These are unused though, as far as I can see. Did you manage to reproduce them?

File: effects/emitters/disintegrator_muzzle_flash_01_emit.bp [hunk 12]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = true,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 150.00,
```
Comment (Garanas) [inline effects/emitters/disintegrator_muzzle_flash_01_emit.bp:14]: Cybran preference here - uef t1 terran artillery has 100 LOD cutoff for muzzle flash 😄

File: effects/emitters/disruptor_hit_shield_emit.bp [hunk 13]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 150.00,
```
Comment (Garanas) [inline effects/emitters/disruptor_hit_shield_emit.bp:14]: Given that a shield can't sustain more than 4 - 8 instances, we can pump this up to 300.

File: effects/emitters/economy_electricity_01_emit.bp [hunk 14]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = true,
-	LODCutoff = -1.00,
+	LODCutoff = 150.00,
```
Comment (Garanas) [inline effects/emitters/economy_electricity_01_emit.bp:14]: of all effects - **this** effect was always rendered!

File: effects/emitters/electron_burst_cloud_02_emit.bp [hunk 15]
```diff
@@ -1,3 +1,4 @@
+-- not used by faf
```
Comment (Garanas) [inline effects/emitters/electron_burst_cloud_02_emit.bp:1]: these are interesting to remember, perhaps we can do some fancy effect when a power generator gets hit (t2 / t3)

File: effects/emitters/empfluxwarhead_01_emit.bp [hunk 16]
```diff
@@ -10,7 +11,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = -1.00, -- acu death effect
```
Comment (Garanas) [inline effects/emitters/empfluxwarhead_01_emit.bp:14]: all good - last one to review for today.

**Comment by Garanas** [comment]
huh, I do not understand how this got merged 🤔 ?

I did a local commit to the branch.

File: effects/emitters/destruction_explosion_fire_shadow_01_emit.bp (hunk 11 above)
Comment (MadMaxFAF) [reply effects/emitters/destruction_explosion_fire_shadow_01_emit.bp:1]: i just copied from the other linked effects

**Pull request merged**